src/lib.rs

```diff
@@ -698,14 +698,19 @@ impl AstarteSdk {
             let data: AstarteType = data.into();
 
             let interfaces = self.interfaces.read().await;
+            let interface_major = interfaces
+                .interfaces
+                .get(interface_name)
+                .unwrap()
```

Comment:
Maybe is better to propagate the error with `?` than generate a panic in case of error or empty value.